<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="7" ignored="4" total="29" passed="18">
  <reporter-output>
  </reporter-output>
  <suite name="TestSuite" duration-ms="39392" started-at="2018-08-03T19:09:17Z" finished-at="2018-08-03T19:09:57Z">
    <groups>
    </groups>
    <test name="Testing" duration-ms="39392" started-at="2018-08-03T19:09:17Z" finished-at="2018-08-03T19:09:57Z">
      <class name="tests.LoginTest">
        <test-method status="PASS" signature="beforeClass()[pri:0, instance:tests.LoginTest@18be83e4]" name="beforeClass" is-config="true" duration-ms="4292" started-at="2018-08-03T19:09:13Z" finished-at="2018-08-03T19:09:17Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method status="PASS" signature="correctEmailCorrectPassword()[pri:14, instance:tests.LoginTest@18be83e4]" name="correctEmailCorrectPassword" duration-ms="4000" started-at="2018-08-03T19:09:17Z" finished-at="2018-08-03T19:09:21Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- correctEmailCorrectPassword -->
        <test-method status="PASS" signature="afterClass()[pri:0, instance:tests.LoginTest@18be83e4]" name="afterClass" is-config="true" duration-ms="2308" started-at="2018-08-03T19:09:57Z" finished-at="2018-08-03T19:09:59Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterClass -->
      </class> <!-- tests.LoginTest -->
      <class name="tests.InvoicesListTest">
        <test-method status="PASS" signature="userID()[pri:31, instance:tests.InvoicesListTest@4b9e13df]" name="userID" duration-ms="59" started-at="2018-08-03T19:09:39Z" finished-at="2018-08-03T19:09:39Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- userID -->
        <test-method status="FAIL" signature="filterClient()[pri:32, instance:tests.InvoicesListTest@4b9e13df]" name="filterClient" duration-ms="2008" started-at="2018-08-03T19:09:39Z" finished-at="2018-08-03T19:09:41Z">
          <exception class="org.openqa.selenium.WebDriverException">
            <message>
              <![CDATA[unknown error: Element <label for="filter-client" class="v-label" style="left: 0px; right: auto; position: absolute;">...</label> is not clickable at point (398, 224). Other element would receive the click: <div class="v-select__selections">...</div>
  (Session info: chrome=67.0.3396.99)
  (Driver info: chromedriver=2.40.565498 (ea082db3280dd6843ebfb08a625e3eb905c4f5ab),platform=Windows NT 6.1.7601 SP1 x86_64) (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 0 milliseconds
Build info: version: '3.13.0', revision: '2f0d292', time: '2018-06-25T15:32:14.902Z'
System info: host: 'KRSTA-PC', ip: '192.168.23.44', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.8.0_171'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, acceptSslCerts: false, applicationCacheEnabled: false, browserConnectionEnabled: false, browserName: chrome, chrome: {chromedriverVersion: 2.40.565498 (ea082db3280dd6..., userDataDir: C:\Users\Krsta\AppData\Loca...}, cssSelectorsEnabled: true, databaseEnabled: false, handlesAlerts: true, hasTouchScreen: false, javascriptEnabled: true, locationContextEnabled: true, mobileEmulationEnabled: false, nativeEvents: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: XP, platformName: XP, rotatable: false, setWindowRect: true, takesHeapSnapshot: true, takesScreenshot: true, unexpectedAlertBehaviour: , unhandledPromptBehavior: , version: 67.0.3396.99, webStorageEnabled: true}
Session ID: 13e481db5c7b71f2e7fe9046d4d6ce06]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.WebDriverException: unknown error: Element <label for="filter-client" class="v-label" style="left: 0px; right: auto; position: absolute;">...</label> is not clickable at point (398, 224). Other element would receive the click: <div class="v-select__selections">...</div>
  (Session info: chrome=67.0.3396.99)
  (Driver info: chromedriver=2.40.565498 (ea082db3280dd6843ebfb08a625e3eb905c4f5ab),platform=Windows NT 6.1.7601 SP1 x86_64) (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 0 milliseconds
Build info: version: '3.13.0', revision: '2f0d292', time: '2018-06-25T15:32:14.902Z'
System info: host: 'KRSTA-PC', ip: '192.168.23.44', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.8.0_171'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, acceptSslCerts: false, applicationCacheEnabled: false, browserConnectionEnabled: false, browserName: chrome, chrome: {chromedriverVersion: 2.40.565498 (ea082db3280dd6..., userDataDir: C:\Users\Krsta\AppData\Loca...}, cssSelectorsEnabled: true, databaseEnabled: false, handlesAlerts: true, hasTouchScreen: false, javascriptEnabled: true, locationContextEnabled: true, mobileEmulationEnabled: false, nativeEvents: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: XP, platformName: XP, rotatable: false, setWindowRect: true, takesHeapSnapshot: true, takesScreenshot: true, unexpectedAlertBehaviour: , unhandledPromptBehavior: , version: 67.0.3396.99, webStorageEnabled: true}
Session ID: 13e481db5c7b71f2e7fe9046d4d6ce06
at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
at java.lang.reflect.Constructor.newInstance(Unknown Source)
at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:214)
at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:166)
at org.openqa.selenium.remote.http.JsonHttpResponseCodec.reconstructValue(JsonHttpResponseCodec.java:40)
at org.openqa.selenium.remote.http.AbstractHttpResponseCodec.decode(AbstractHttpResponseCodec.java:80)
at org.openqa.selenium.remote.http.AbstractHttpResponseCodec.decode(AbstractHttpResponseCodec.java:44)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:158)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:548)
at org.openqa.selenium.remote.RemoteWebElement.execute(RemoteWebElement.java:276)
at org.openqa.selenium.remote.RemoteWebElement.click(RemoteWebElement.java:83)
at sun.reflect.GeneratedMethodAccessor2.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:51)
at com.sun.proxy.$Proxy9.click(Unknown Source)
at pages.InvoicesPage.filterClient(InvoicesPage.java:104)
at tests.InvoicesListTest.filterClient(InvoicesListTest.java:22)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.WebDriverException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- filterClient -->
        <test-method status="FAIL" signature="thisMonth()[pri:33, instance:tests.InvoicesListTest@4b9e13df]" name="thisMonth" duration-ms="1168" started-at="2018-08-03T19:09:41Z" finished-at="2018-08-03T19:09:42Z">
          <exception class="java.lang.AssertionError">
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError
at org.junit.Assert.fail(Assert.java:86)
at org.junit.Assert.assertTrue(Assert.java:41)
at org.junit.Assert.assertTrue(Assert.java:52)
at tests.InvoicesListTest.thisMonth(InvoicesListTest.java:48)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- thisMonth -->
        <test-method status="FAIL" signature="lastMonth()[pri:34, instance:tests.InvoicesListTest@4b9e13df]" name="lastMonth" duration-ms="986" started-at="2018-08-03T19:09:42Z" finished-at="2018-08-03T19:09:43Z">
          <exception class="java.lang.AssertionError">
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError
at org.junit.Assert.fail(Assert.java:86)
at org.junit.Assert.assertTrue(Assert.java:41)
at org.junit.Assert.assertTrue(Assert.java:52)
at tests.InvoicesListTest.lastMonth(InvoicesListTest.java:66)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- lastMonth -->
        <test-method status="FAIL" signature="resetMonth()[pri:35, instance:tests.InvoicesListTest@4b9e13df]" name="resetMonth" duration-ms="1325" started-at="2018-08-03T19:09:43Z" finished-at="2018-08-03T19:09:44Z">
          <exception class="java.lang.AssertionError">
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError
at org.junit.Assert.fail(Assert.java:86)
at org.junit.Assert.assertTrue(Assert.java:41)
at org.junit.Assert.assertTrue(Assert.java:52)
at tests.InvoicesListTest.resetMonth(InvoicesListTest.java:84)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- resetMonth -->
        <test-method status="FAIL" signature="resetInvoices()[pri:36, instance:tests.InvoicesListTest@4b9e13df]" name="resetInvoices" duration-ms="554" started-at="2018-08-03T19:09:44Z" finished-at="2018-08-03T19:09:45Z">
          <exception class="java.lang.AssertionError">
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError
at org.junit.Assert.fail(Assert.java:86)
at org.junit.Assert.assertTrue(Assert.java:41)
at org.junit.Assert.assertTrue(Assert.java:52)
at tests.InvoicesListTest.resetInvoices(InvoicesListTest.java:101)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- resetInvoices -->
        <test-method status="FAIL" signature="numOfInvoices()[pri:37, instance:tests.InvoicesListTest@4b9e13df]" name="numOfInvoices" duration-ms="76" started-at="2018-08-03T19:09:45Z" finished-at="2018-08-03T19:09:45Z">
          <exception class="java.lang.AssertionError">
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError
at org.junit.Assert.fail(Assert.java:86)
at org.junit.Assert.assertTrue(Assert.java:41)
at org.junit.Assert.assertTrue(Assert.java:52)
at tests.InvoicesListTest.numOfInvoices(InvoicesListTest.java:116)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- numOfInvoices -->
        <test-method status="FAIL" signature="previewInvoice()[pri:38, instance:tests.InvoicesListTest@4b9e13df]" name="previewInvoice" duration-ms="11517" started-at="2018-08-03T19:09:45Z" finished-at="2018-08-03T19:09:57Z">
          <exception class="org.openqa.selenium.TimeoutException">
            <message>
              <![CDATA[Expected condition failed: waiting for visibility of [[ChromeDriver: chrome on XP (13e481db5c7b71f2e7fe9046d4d6ce06)] -> xpath: //*[text()[contains(.,'Do you want to delete this invoice?')]]] (tried for 10 second(s) with 500 milliseconds interval)
Build info: version: '3.13.0', revision: '2f0d292', time: '2018-06-25T15:32:14.902Z'
System info: host: 'KRSTA-PC', ip: '192.168.23.44', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.8.0_171'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, acceptSslCerts: false, applicationCacheEnabled: false, browserConnectionEnabled: false, browserName: chrome, chrome: {chromedriverVersion: 2.40.565498 (ea082db3280dd6..., userDataDir: C:\Users\Krsta\AppData\Loca...}, cssSelectorsEnabled: true, databaseEnabled: false, handlesAlerts: true, hasTouchScreen: false, javascriptEnabled: true, locationContextEnabled: true, mobileEmulationEnabled: false, nativeEvents: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: XP, platformName: XP, rotatable: false, setWindowRect: true, takesHeapSnapshot: true, takesScreenshot: true, unexpectedAlertBehaviour: , unhandledPromptBehavior: , version: 67.0.3396.99, webStorageEnabled: true}
Session ID: 13e481db5c7b71f2e7fe9046d4d6ce06]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.TimeoutException: Expected condition failed: waiting for visibility of [[ChromeDriver: chrome on XP (13e481db5c7b71f2e7fe9046d4d6ce06)] -> xpath: //*[text()[contains(.,'Do you want to delete this invoice?')]]] (tried for 10 second(s) with 500 milliseconds interval)
Build info: version: '3.13.0', revision: '2f0d292', time: '2018-06-25T15:32:14.902Z'
System info: host: 'KRSTA-PC', ip: '192.168.23.44', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.8.0_171'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, acceptSslCerts: false, applicationCacheEnabled: false, browserConnectionEnabled: false, browserName: chrome, chrome: {chromedriverVersion: 2.40.565498 (ea082db3280dd6..., userDataDir: C:\Users\Krsta\AppData\Loca...}, cssSelectorsEnabled: true, databaseEnabled: false, handlesAlerts: true, hasTouchScreen: false, javascriptEnabled: true, locationContextEnabled: true, mobileEmulationEnabled: false, nativeEvents: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: XP, platformName: XP, rotatable: false, setWindowRect: true, takesHeapSnapshot: true, takesScreenshot: true, unexpectedAlertBehaviour: , unhandledPromptBehavior: , version: 67.0.3396.99, webStorageEnabled: true}
Session ID: 13e481db5c7b71f2e7fe9046d4d6ce06
at org.openqa.selenium.support.ui.WebDriverWait.timeoutException(WebDriverWait.java:81)
at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:271)
at pages.InvoicesPage.deleteMsg(InvoicesPage.java:198)
at tests.InvoicesListTest.previewInvoice(InvoicesListTest.java:130)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.TimeoutException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- previewInvoice -->
      </class> <!-- tests.InvoicesListTest -->
      <class name="tests.AddNewInvoiceTest">
        <test-method status="PASS" signature="addNewInvoiceAndGoBack()[pri:15, instance:tests.AddNewInvoiceTest@cb5822]" name="addNewInvoiceAndGoBack" duration-ms="961" started-at="2018-08-03T19:09:21Z" finished-at="2018-08-03T19:09:22Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addNewInvoiceAndGoBack -->
        <test-method status="PASS" signature="addNewInvoice()[pri:16, instance:tests.AddNewInvoiceTest@cb5822]" name="addNewInvoice" duration-ms="844" started-at="2018-08-03T19:09:22Z" finished-at="2018-08-03T19:09:23Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addNewInvoice -->
        <test-method status="PASS" signature="noInputJustSave()[pri:17, instance:tests.AddNewInvoiceTest@cb5822]" name="noInputJustSave" duration-ms="1274" started-at="2018-08-03T19:09:23Z" finished-at="2018-08-03T19:09:24Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- noInputJustSave -->
        <test-method status="PASS" signature="CheckNewInvoiceForm()[pri:18, instance:tests.AddNewInvoiceTest@cb5822]" name="CheckNewInvoiceForm" duration-ms="76" started-at="2018-08-03T19:09:24Z" finished-at="2018-08-03T19:09:24Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CheckNewInvoiceForm -->
        <test-method status="PASS" signature="formStatus()[pri:19, instance:tests.AddNewInvoiceTest@cb5822]" name="formStatus" duration-ms="753" started-at="2018-08-03T19:09:24Z" finished-at="2018-08-03T19:09:25Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- formStatus -->
        <test-method status="PASS" signature="to()[pri:20, instance:tests.AddNewInvoiceTest@cb5822]" name="to" duration-ms="853" started-at="2018-08-03T19:09:25Z" finished-at="2018-08-03T19:09:26Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- to -->
        <test-method status="PASS" signature="from()[pri:21, instance:tests.AddNewInvoiceTest@cb5822]" name="from" duration-ms="914" started-at="2018-08-03T19:09:26Z" finished-at="2018-08-03T19:09:27Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- from -->
        <test-method status="PASS" signature="submittedDate()[pri:22, instance:tests.AddNewInvoiceTest@cb5822]" name="submittedDate" duration-ms="1001" started-at="2018-08-03T19:09:27Z" finished-at="2018-08-03T19:09:28Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- submittedDate -->
        <test-method status="PASS" signature="dueDate()[pri:23, instance:tests.AddNewInvoiceTest@cb5822]" name="dueDate" duration-ms="2038" started-at="2018-08-03T19:09:28Z" finished-at="2018-08-03T19:09:30Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- dueDate -->
        <test-method status="PASS" signature="basis()[pri:24, instance:tests.AddNewInvoiceTest@cb5822]" name="basis" duration-ms="531" started-at="2018-08-03T19:09:30Z" finished-at="2018-08-03T19:09:30Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- basis -->
        <test-method status="PASS" signature="toCurrencyPicked()[pri:25, instance:tests.AddNewInvoiceTest@cb5822]" name="toCurrencyPicked" duration-ms="243" started-at="2018-08-03T19:09:30Z" finished-at="2018-08-03T19:09:31Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- toCurrencyPicked -->
        <test-method status="PASS" signature="addWrongInvoiceContentAndDelete()[pri:26, instance:tests.AddNewInvoiceTest@cb5822]" name="addWrongInvoiceContentAndDelete" duration-ms="3331" started-at="2018-08-03T19:09:31Z" finished-at="2018-08-03T19:09:34Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addWrongInvoiceContentAndDelete -->
        <test-method status="PASS" signature="addInvoiceContent()[pri:27, instance:tests.AddNewInvoiceTest@cb5822]" name="addInvoiceContent" duration-ms="2326" started-at="2018-08-03T19:09:34Z" finished-at="2018-08-03T19:09:36Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addInvoiceContent -->
        <test-method status="PASS" signature="bank()[pri:28, instance:tests.AddNewInvoiceTest@cb5822]" name="bank" duration-ms="880" started-at="2018-08-03T19:09:36Z" finished-at="2018-08-03T19:09:37Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- bank -->
        <test-method status="PASS" signature="save()[pri:29, instance:tests.AddNewInvoiceTest@cb5822]" name="save" duration-ms="1050" started-at="2018-08-03T19:09:37Z" finished-at="2018-08-03T19:09:38Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- save -->
        <test-method status="PASS" signature="title()[pri:30, instance:tests.AddNewInvoiceTest@cb5822]" name="title" duration-ms="557" started-at="2018-08-03T19:09:38Z" finished-at="2018-08-03T19:09:39Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- title -->
      </class> <!-- tests.AddNewInvoiceTest -->
    </test> <!-- Testing -->
  </suite> <!-- TestSuite -->
</testng-results>
